doc/source/how-to/manage-protobuf-definitions.rst

```diff
@@ -0,0 +1,163 @@
+Manage Protobuf definitions
+===========================
+
+Protobuf service definitions provide the API specification for underlying
+server implementations, so that each consuming client library has a clear
+contract for gRPC data messages. Ideally, the .proto files have a single
+repository established as the source of truth for the .proto files,
```

Comment:
```suggestion
repository established as the source of truth for the ``.proto`` files,
```

Reply:
Would this sentence be better written as follows:

Ideally, a single repository is established as the source of truth for the PROTO files.

Reply:
Better naming them as ``.proto``